sb3_contrib/common/torch_layers.py

```diff
@@ -0,0 +1,117 @@
+import torch
+
+__all__ = ["BatchRenorm1d", "BatchRenorm"]
+
+
+class BatchRenorm(torch.jit.ScriptModule):
```

Comment:
btw, why did you choose to use a `ScriptModule`, is it faster? if so, how much?

Reply:
tbh, I don't remember, why I used it. And I just found that it seems to have been deprecated in a while anyway, so we should probably just use nn.Module instead (https://discuss.pytorch.org/t/whats-the-difference-between-torch-nn-module-and-torch-jit-scriptmodule/64480)